src/domain/enrolment/CreateEnrolmentPage.tsx

```diff
@@ -46,7 +46,7 @@ const CreateEnrolmentPage: React.FC = () => {
   const { data: eventData, loading } = useEventQuery({
     variables: {
       id: eventId as string,
-      include: ['location', 'keywords'],
+      include: ['keywords', 'location'],
```

Comment:
I think I have had the same issue. Best way to avoid wasting time is to just put those variables to JSON diff right away if there seem to be problems with graphql mocks

Reply:
Yes the json diff did the trick to find the problem